paper/main.tex

```diff
@@ -706,369 +712,343 @@ \section{Evaluation of performance}\label{section:evaluation_of_performance}
 tournaments the higher the rates of cooperation the lower a strategy's success
 and median score. A strategy would want to cooperate less than both
 the mean and median cooperator in such settings. In probabilistic ending
-tournaments the correlations coefficients have a larger values, indicating a
-stronger effect. Thus a strategy will be punished more by it's cooperative
-behaviour in probabilistic ending environments, this was seen in Section~\ref{section:evaluation_of_performance}
+tournaments the correlation coefficients have larger values, indicating a
+stronger effect. Thus a strategy will be punished more by its cooperative
+behaviour in probabilistic ending environments, supporting the results of
+Section~\ref{section:evaluation_of_performance}
 as well. The distributions of the $C_r$ of the winners in
-both tournaments is given by Figure~\ref{fig:c_r_distributions}. It confirms
-that the winners in noisy tournaments cooperated less than 35\% of the times
+both tournaments are given by Figure~\ref{fig:c_r_distributions}. It confirms
+that the winners in noisy tournaments cooperated less than 35\% of the time
 and in probabilistic ending tournaments less than 10\%.
-In noisy probabilistic ending tournaments and in over all the tournaments' results,
-the only features that had a moderate affect are $C_r/C_{\text{mean}},
+In noisy probabilistic ending tournaments and over all the tournaments' results,
+the only features that had a moderate effect are $C_r/C_{\text{mean}},
 C_r/C_{\text{max}}$ and $C_r$. In such environments cooperative behaviour
-appears to be punished by not as much as in noisy and probabilistic ending
+appears to be punished less than in noisy and probabilistic ending
 tournaments.
 
+Moreover, the manner in which a strategy achieves a given cooperation rate
+relative to the tournament population average is important. Playing a strategy
+that randomly cooperates with $C_{\text{mean}}$ is unlikely to be as effective as it will
+randomly have cooperations matched with defections. In contrast, TFT naturally
+achieves a cooperation rate near $C_{\text{mean}}$ by virtue of copying its opponent's
+last move while also minimizing instances where it is exploited by an opponent
+(cooperating while the opponent defects), at least in non-noisy tournaments.
+\footnote{This also explains why Tit For N Tats does not fare well -- it fails
+to achieve the proper cooperation ratio.}
+TFT forces the opponent to pay back the (C, D) round with a (D, C) round before
```

Comment:
We need to be consistent in presentation of this. @Nikoleta-v3 does this need to be `\((C, D)\) round with a \((D, C)\)`.

(I don't care what we do, let's just make sure we're consistant).